__tests__/integration/txLogger.js

```diff
@@ -0,0 +1,137 @@
+import * as Path from "path";
+
+const fsold = require('fs');
+const fs = fsold.promises;
+
+export const loggers = {
+  /**
+   * @type: TxLogger
+   */
+  test: null
+}
+
+/**
+ * A logger for every transaction on the integration tests for debugging.
+ */
+export class TxLogger {
+  #instanceFilename;
+  #fileFullPath;
+
+  get filename() {
+    return this.#instanceFilename;
+  }
+
+  /**
+   * Builds the log filename based on current time and an optional title
+   * @param {string} [title] Optional title. Keep it short and simple for readability
+   */
+  constructor(title) {
+    const date = new Date();
+    const timestamp = date.toISOString()
+      .replace(/-/g,'') // Remove date separator
+      .replace(/:/g,'') // Remove hour separator
+      .split('.')[0] // Get only the seconds integer
+    const additionalTitle = `-${title}` || ''
```

Comment:
Why do you have this `||`? I think it will never get to it.

Reply:
Fixed the logic with a ternary operator.
If the `title` is not informed, the resulting filename will be something like `20220224T084737-integrationTest.log`